src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  margin-right: 48px;
+  margin-bottom: 46px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    align-self: center;
+  }
+
+  &__product-name {
+    margin: 40px 0 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-main;
+  }
+
+  &__product-code {
+    font-family: "Roboto", sans-serif;
```

Comment:
```suggestion
```
redundant . almost settled in .card.
and better to set main font-family in html selector